[2024-11-14T00:12:53.681+0000] {processor.py:186} INFO - Started process (PID=43) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:12:53.682+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:12:53.685+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:12:53.684+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:12:53.699+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:12:53.695+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:12:53.700+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:12:53.727+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.050 seconds
[2024-11-14T00:13:23.859+0000] {processor.py:186} INFO - Started process (PID=107) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:13:23.860+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:13:23.862+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:13:23.862+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:13:23.866+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:13:23.865+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:13:23.867+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:13:23.878+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.026 seconds
[2024-11-14T00:13:54.543+0000] {processor.py:186} INFO - Started process (PID=171) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:13:54.544+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:13:54.546+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:13:54.546+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:13:54.549+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:13:54.549+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:13:54.550+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:13:54.563+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.024 seconds
[2024-11-14T00:14:25.046+0000] {processor.py:186} INFO - Started process (PID=235) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:14:25.046+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:14:25.048+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:14:25.048+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:14:25.050+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:14:25.050+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:14:25.051+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:14:25.060+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.020 seconds
[2024-11-14T00:14:55.502+0000] {processor.py:186} INFO - Started process (PID=299) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:14:55.502+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:14:55.503+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:14:55.503+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:14:55.506+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:14:55.506+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:14:55.507+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:14:55.514+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.015 seconds
[2024-11-14T00:15:25.837+0000] {processor.py:186} INFO - Started process (PID=363) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:15:25.839+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:15:25.841+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:15:25.841+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:15:25.847+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:15:25.846+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:15:25.847+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:15:25.860+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.029 seconds
[2024-11-14T00:15:56.339+0000] {processor.py:186} INFO - Started process (PID=427) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:15:56.340+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:15:56.344+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:15:56.343+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:15:56.350+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:15:56.349+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:15:56.350+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:15:56.366+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.033 seconds
[2024-11-14T00:16:26.870+0000] {processor.py:186} INFO - Started process (PID=491) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:16:26.870+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:16:26.873+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:16:26.873+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:16:26.878+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:16:26.877+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:16:26.878+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:16:26.891+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.028 seconds
[2024-11-14T00:16:57.358+0000] {processor.py:186} INFO - Started process (PID=555) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:16:57.359+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:16:57.361+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:16:57.361+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:16:57.365+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:16:57.364+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:16:57.366+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:16:57.380+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.028 seconds
[2024-11-14T00:17:27.639+0000] {processor.py:186} INFO - Started process (PID=619) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:17:27.640+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:17:27.642+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:17:27.641+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:17:27.646+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:17:27.645+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:17:27.646+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:17:27.659+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.027 seconds
[2024-11-14T00:17:58.269+0000] {processor.py:186} INFO - Started process (PID=684) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:17:58.269+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:17:58.271+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:17:58.271+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:17:58.275+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:17:58.274+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:17:58.275+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:17:58.290+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.027 seconds
[2024-11-14T00:18:28.635+0000] {processor.py:186} INFO - Started process (PID=750) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:18:28.637+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:18:28.639+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:18:28.639+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:18:28.643+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:18:28.642+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:18:28.644+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:18:28.655+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.026 seconds
[2024-11-14T00:18:59.023+0000] {processor.py:186} INFO - Started process (PID=821) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:18:59.023+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:18:59.025+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:18:59.025+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:18:59.028+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:18:59.027+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:18:59.029+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:18:59.043+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.026 seconds
[2024-11-14T00:23:09.416+0000] {processor.py:186} INFO - Started process (PID=36) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:23:09.416+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:23:09.420+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:23:09.420+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:23:09.427+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:23:09.425+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:23:09.430+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:23:09.449+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.038 seconds
[2024-11-14T00:23:39.551+0000] {processor.py:186} INFO - Started process (PID=101) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:23:39.553+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:23:39.554+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:23:39.554+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:23:39.560+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:23:39.557+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:23:39.561+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:23:39.572+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.024 seconds
[2024-11-14T00:24:10.000+0000] {processor.py:186} INFO - Started process (PID=166) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:24:10.001+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:24:10.004+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:24:10.003+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:24:10.010+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:24:10.006+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:24:10.010+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:24:10.025+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.028 seconds
[2024-11-14T00:24:40.191+0000] {processor.py:186} INFO - Started process (PID=231) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:24:40.192+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:24:40.194+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:24:40.194+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:24:40.201+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:24:40.200+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:24:40.202+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:24:40.218+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.032 seconds
[2024-11-14T00:25:10.675+0000] {processor.py:186} INFO - Started process (PID=296) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:25:10.676+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:25:10.678+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:25:10.678+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:25:10.686+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:25:10.682+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:25:10.687+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:25:10.706+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.036 seconds
[2024-11-14T00:25:41.211+0000] {processor.py:186} INFO - Started process (PID=361) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:25:41.211+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:25:41.214+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:25:41.213+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:25:41.221+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:25:41.217+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:25:41.222+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:25:41.234+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.027 seconds
[2024-11-14T00:26:11.533+0000] {processor.py:186} INFO - Started process (PID=426) to work on /opt/airflow/dags/collect_dag.py
[2024-11-14T00:26:11.535+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/collect_dag.py for tasks to queue
[2024-11-14T00:26:11.538+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:26:11.537+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:26:11.543+0000] {logging_mixin.py:190} INFO - [2024-11-14T00:26:11.542+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/collect_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_dag.py", line 9, in <module>
    from .collect import extract_en_species
ImportError: attempted relative import with no known parent package
[2024-11-14T00:26:11.545+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_dag.py
[2024-11-14T00:26:11.566+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/collect_dag.py took 0.038 seconds
